.github/workflows/build.yml

```diff
@@ -23,16 +22,18 @@ jobs:
           - python3
           - sed
         include:
-          - container: debian7
+          - arch: x86_64
+            runs-on: ubuntu-latest
+          # TODO: add aarch64 when available
```

Comment:
Sorry, I'd rather we didn't add `aarch64` in here until we have CI to test it (and same for Portable Ruby).